packages/@webex/plugin-meetings/test/unit/spec/meeting/request.js

```diff
@@ -195,7 +197,42 @@ describe('plugin-meetings', () => {
         assert.equal(requestParams.body.device.countryCode, 'US');
         assert.equal(requestParams.body.permissionToken, 'permission-token');
         assert.equal(requestParams.body.device.regionCode, 'WEST-COAST');
-      });
+        assert.include(requestParams.body.device.localIp, '127.0.0');
+      });
+
+      describe('clientAddress geoHintInfo undefined', () => {
+        beforeEach(() => {
+          webex.meetings.geoHintInfo = {};
+        });
+    
+        // reset
+        afterEach(() => {
+          webex.meetings.geoHintInfo = {
+            countryCode: 'US',
+            regionCode: 'WEST-COAST',
+            clientAddress: '127.0.0.1',
```

Comment:
suggestion: To avoid issues in future when adding more items to the geo hint (if there are others) I suggest changing uses of it to use a constant:

```js
const geoHintInfoDefaults = {countryCode: 'US', regionCode: 'WEST-COAST', clientAddress: '127.0.0.1'};

...

// Wherever it needs setting
webex.meetings.geoHintInfo = {...geoHintInfoDefaults};
```

Makes this a little more DRY and easier to maintain (read: harder to miss) later.

Reply:
yeah thats cool. easy change. Will do.

Reply:
done